src/modules/drawing/components/DrawingStylePopup.vue

```diff
@@ -157,14 +183,21 @@ export default {
                 return this.feature.get('type') === 'TEXT'
             },
         },
+        isFeatureLine: {
+            get() {
+                return this.feature.get('type') === 'LINE'
+            },
+        },
```

Comment:
Here you can simplify the syntax to 
```
isFeatureLine: () => this.feature.get('type') === 'LINE',
```
if you want, same for the others above. Using the `get/set` syntax is good when your computed needs a setter, but if it's read only there's no need to use this more complex syntax.

Reply:
It was not possible to use arrow functions in this case so I simplified the syntax with plain functions.